arenadata/scripts/run_resgroup_test.bash

```diff
@@ -1,28 +1,35 @@
 #!/bin/bash
+set -eox pipefail
 
 project="resgroup"
 
+function cleanup {
+  docker-compose -p $project -f arenadata/docker-compose.yaml --env-file arenadata/.env down
```

Comment:
Do we care if cleanup fails? It can mess up exit code as well.
```suggestion
  docker-compose -p $project -f arenadata/docker-compose.yaml --env-file arenadata/.env down || true
```

Reply:
Looks like exit code of command in trap function does not affect exit code of the script.
example:
```bash
#!/bin/bash
#test.sh

function cleanup () {
	echo "cleanup"
	./exit.sh
}

trap cleanup EXIT

echo "do something"

exit 1

echo "do fine"
```

```bash
#!/bin/bash
#exit.sh

echo "exit"
exit 2
echo "fine"
```

Run `test.sh` and then `echo $?`. You can see that exit code of `test.sh` is 1 while exit code of `exit.sh` is 2.

Reply:
I see, thanks